doc/YubiKey_PIV_introduction.adoc

```diff
@@ -37,16 +37,16 @@ history, this can be avoided by leaving the argument out all-together and the
 software will ask for key/pin to be input. For the management key option (-k)
 this is achieved by leaving out the value but will specifying -k.
 
-  $ key=`dd if=/dev/random bs=1 count=24 2>/dev/null | hexdump -v -e '/1 "%02X"'`
+  $ key=`dd if=/dev/urandom 2>/dev/null | tr -d '[:lower:]' | tr -cd '[:xdigit:]' | fold -w48 | head -1` 
```

Comment:
The `tr` commands will fail on MacOS without `LC_CTYPE=C` set in the environment for those two `tr` invocations.

Reply:
Indeed; I remember running into this problem last year. Does https://github.com/Yubico/yubico-piv-tool/commit/646ea3e4 fix the problem for you?

Reply:
It does. Thanks!